java/memory/src/main/java/org/apache/arrow/memory/AllocationManager.java

```diff
@@ -196,6 +196,10 @@ void release(final BufferLedger ledger) {
     }
   }
 
+  public void releaseMemory() {
```

Comment:
is this change required ? I don't think it should be a public API.

Reply:
my bad, I will revert the changes on AllocationManager.java